UnityProject/Assets/Scripts/Objects/PushPull.cs

```diff
@@ -152,31 +146,37 @@ public void BreakPull()
 		pulledBy = null;
 	}
 
+	//This is to turn off CNT while pulling an object
+	[ClientRpc]
+	public void RpcToggleCnt(bool activeState){
```

Comment:
The whole idea of turning CNT off is very questionable (it's not meant to be turned off), but we'll see how it will perform in practice. 
This can potentially cause problems: what happens if server forces a SetPosition via CNT, but it will be disabled on client's side? (I suspect it will be ignored)

Reply:
Shouldn't it just be integrated with CNT instead of already butchering CNT to get this feature in?